app/controllers/ListEventsController.js

```diff
@@ -9,10 +10,21 @@ GameUpApp.controller('ListEventsController', ['$scope', '$location', function($s
     return query.find();
   };
 
+  $scope.retrievePromotedEventList = function(){
```

Comment:
Good point. I guess we should add a limit to the number of promoted events that are listed. Hopefully though the total number of promoted events wouldn't ever be more than 5 or something.

We'd also have to think about promoted events by region - the promoted events shown in Toronto will be different for the ones in Ottawa. I don't know how we'd work out that logic...